posts/2021/esds-update-september-2021.md

```diff
@@ -0,0 +1,152 @@
+- [Pythia Portal Overview (30 July 2021)](https://ncar.github.io/esds/posts/2021/project-pythia-overview/)
+
+## Python Tutorials
+
+As mentioned previously, the Python Tutorial series has been on ongoing training resource, led by Julia Kent and fellow Xdevers. Below is a list of the tutorial blog posts and associated videos.
+
+#### [Numpy Tutorial (10 March 2021)](https://ncar.github.io/esds/posts/2021/numpy-tutorial/)
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/kstc-6uz7AQ/0.jpg)](https://www.youtube.com/watch?v=kstc-6uz7AQ)
+
+#### [Matplotlib Tutorial (24 March 2021)](https://ncar.github.io/esds/posts/2021/matplotlib-tutorial/)
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/EiPRIdHQEmE/0.jpg)](https://www.youtube.com/watch?v=EiPRIdHQEmE)
+
+#### [Object Oriented Programming Tutorial (09 April 2021)](https://ncar.github.io/esds/posts/2021/object-oriented-programming-tutorial/)
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/GEFnL8C62u8/0.jpg)](https://www.youtube.com/watch?v=GEFnL8C62u8)
+
+#### [Cartopy Tutorial (26 April 2021)](https://ncar.github.io/esds/posts/2021/cartopy-tutorial/)
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/ivmd3RluMiw/0.jpg)](https://www.youtube.com/watch?v=ivmd3RluMiw)
+
+#### [Git and Github Tutorial (12 May 2021)](https://ncar.github.io/esds/posts/2021/git-and-github-tutorial/)
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/fYkPn0Nttlg/0.jpg)](https://www.youtube.com/watch?v=fYkPn0Nttlg)
+
+#### [Pandas Tutorial (26 May 2021)](https://ncar.github.io/esds/posts/2021/pandas-tutorial/)
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/BsV3ek7qsiM/0.jpg)](https://www.youtube.com/watch?v=BsV3ek7qsiM)
+
+#### [Xarray Tutorial (09 and 23 June 2021)](https://ncar.github.io/esds/posts/2021/xarray-tutorial/)
+
+##### Part 1
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/Ss4ryKukhi4/0.jpg)](https://www.youtube.com/watch?v=Ss4ryKukhi4)
+
+##### Part 2
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/2H_4drBwORY/0.jpg)](https://www.youtube.com/watch?v=2H_4drBwORY)
+
+#### [Dask Tutorial (14 July and 11 August 2021)](https://ncar.github.io/esds/posts/2021/dask-tutorial/)
+
+##### Part 1
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/wn-QM6QUB_U/0.jpg)](https://www.youtube.com/watch?v=wn-QM6QUB_U)
+
+##### Part 2
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/yn4_-1pHC5k/0.jpg)](https://www.youtube.com/watch?v=yn4_-1pHC5k)
+
+#### [GeoCAT Viz Tutorial (25 August 2021)](https://ncar.github.io/esds/posts/2021/geocat-tutorial/)
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/It231le1fAU/0.jpg)](https://www.youtube.com/watch?v=It231le1fAU)
+
+#### [GeoCAT Comp Tutorial (08 September 2021)](https://ncar.github.io/esds/posts/2021/geocat-comp-tutorial/)
+
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/uiWDQKI8YTQ/0.jpg)](https://www.youtube.com/watch?v=uiWDQKI8YTQ)
```

Comment:
How about embedding the videos? This is a personal preference though :) 
So, no worries if you prefer linking to the videos instead 😀
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/uiWDQKI8YTQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Reply:
Ohh I like that idea... I will go head and just accordingly!